2025-03-16 21:18:34,917 INFO    MainThread:31900 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-16 21:18:34,917 INFO    MainThread:31900 [wandb_setup.py:_flush():67] Configure stats pid to 31900
2025-03-16 21:18:34,917 INFO    MainThread:31900 [wandb_setup.py:_flush():67] Loading settings from C:\Users\DELL\.config\wandb\settings
2025-03-16 21:18:34,917 INFO    MainThread:31900 [wandb_setup.py:_flush():67] Loading settings from C:\Users\DELL\Documents\SEMESTER 8\Deep Learning\Assignment_1\wandb\settings
2025-03-16 21:18:34,917 INFO    MainThread:31900 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-16 21:18:34,917 INFO    MainThread:31900 [wandb_init.py:setup_run_log_directory():647] Logging user logs to C:\Users\DELL\Documents\SEMESTER 8\Deep Learning\Assignment_1\wandb\run-20250316_211834-5jpz7zht\logs\debug.log
2025-03-16 21:18:34,917 INFO    MainThread:31900 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to C:\Users\DELL\Documents\SEMESTER 8\Deep Learning\Assignment_1\wandb\run-20250316_211834-5jpz7zht\logs\debug-internal.log
2025-03-16 21:18:34,917 INFO    MainThread:31900 [wandb_init.py:init():761] calling init triggers
2025-03-16 21:18:34,918 INFO    MainThread:31900 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-16 21:18:34,918 INFO    MainThread:31900 [wandb_init.py:init():784] starting backend
2025-03-16 21:18:34,918 INFO    MainThread:31900 [wandb_init.py:init():788] sending inform_init request
2025-03-16 21:18:34,956 INFO    MainThread:31900 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2025-03-16 21:18:34,957 INFO    MainThread:31900 [wandb_init.py:init():798] backend started and connected
2025-03-16 21:18:34,958 INFO    MainThread:31900 [wandb_init.py:init():891] updated telemetry
2025-03-16 21:18:34,962 INFO    MainThread:31900 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-16 21:18:35,382 ERROR   MainThread:31900 [wandb_init.py:init():1478] error in wandb.init()
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Roaming\Python\Python312\site-packages\wandb\sdk\wandb_init.py", line 1468, in init
    return wi.init(run_settings, run_config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\DELL\AppData\Roaming\Python\Python312\site-packages\wandb\sdk\wandb_init.py", line 963, in init
    raise error
wandb.errors.errors.CommError: failed to upsert bucket: returned error 403: {"data":{"upsertBucket":null},"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}]}
2025-03-16 21:18:36,388 INFO    MsgRouterThr:31900 [mailbox.py:close():129] Closing mailbox, abandoning 0 handles.
